index.bs

```diff
@@ -5539,10 +5547,12 @@ partial interface MLGraphBuilder {
   </summary>
     1. If [=MLGraphBuilder/validating operand=] with [=this=] and |input| returns false, then [=exception/throw=] a {{TypeError}}.
     1. Let |axis| be |options|.{{MLSplitOptions/axis}}.
+    1. If |axis| is greater than or equal to |input|'s [=MLOperand/rank=], then [=exception/throw=] a {{TypeError}}.
     1. If |splits| is an {{unsigned long}}:
         1. If |input|'s [=MLOperand/shape=][|axis|] % |splits| is not 0, then [=exception/throw=] a {{TypeError}}.
         1. Otherwise, let |splitCount| be |splits|.
     1. If |splits| is a sequence of {{unsigned long}}:
+        1. If any of its elements is equal to 0, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Should we also mention #391 here?

Reply:
Makes sense - I'll add a note here too.

Reply:
Added in 962ca97be0adc271367cce16c2ff8867edad9c9b